R/eastafrica.R

```diff
@@ -0,0 +1,68 @@
+#' get KITE East Africa v2.1.csv from ...
+#'
+#' See:
+#' - A Database of Radiocarbon Dates for Palaeoenvironmental Research in Eastern Africa: https://www.openquaternary.com/articles/10.5334/oq.22/
+#' - Radiocarbon dates from eastern Africa in the CARD2.0 format: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/NJLNRJ
+#'
+#' The dataset is stored within https://dataverse.harvard.edu that requires the dataverse-package (see https://cran.r-project.org/web/packages/dataverse/vignettes/C-retrieval.html)
+#'
+#' @examples
+#'
+#' \dontrun{
+#'   KITEeastAfrica <- get_KITEeastAfrica()
+#' }
+#'
+#' @export
+get_KITEeastAfrica <- function() {
+
+  Sys.setenv("DATAVERSE_SERVER" = "dataverse.harvard.edu")
+
+  writeBin(get_file("CARD Upload Template - KITE East Africa v2.1.csv", "doi:10.7910/DVN/NJLNRJ"), "data-raw/KITE_EastAfrica.csv")
```

Comment:
Please work with a temporary file like I did [here](https://github.com/ISAAKiel/c14bazAAR/blob/4e34cf0a26057916f8cf763ea399c408b2dc71e7/R/14sea.R#L14-L25) and add the internet connection check. 

c14bazAAR should not contain any data by itself. This is not a data- but an API package.